### DISCUSSION TITLE
Guidelines and Principles for PR review

### DISCUSSION THREAD

**Discussion by mihai-sysbio** [SysBioChalmers/Human-GEM#727] [Ideas]
It would be nice to have guidelines in place about the PR review process, more on the lines of how to "run" a review, similar to the guide on how to contribute:
- start reading the top PR message and re-familiarize yourself with the issues that it references and aims to solve
- continue reading the PR discussions
- go through the code changes and make sure these make sense, and use a desktop diff if the online diff doesn't work
- always be appreciative and supportive
- if you have any suggestions, leave them as in-line comments so that they sit closer to the proposed code changes, and summarize these in the final review message
- make sure everything is okay with the GH actions
- who does the merging the reviewer or the author? my suggestion is to allow some days for the author to do this, if not then go ahead
- make sure the target issue is closed automatically, otherwise close it by hand
- check if new contributors should be added, either for the issue or the PR, and ask for their orcid
- if there are any new issues that came up during the discussion, make sure to report these

Any thoughts/ideas?

**Comment by haowang-bioinfo** [comment]
thanks for bring up this, which is good for long-term development and should be better formulated

**Comment by haowang-bioinfo** [comment]
Perhaps let's expand this discussion with some principles:


**Principles for reviewers:**
- Dedication to truth
- Practice independent and critical thinking
- Be open and inclusive, with [growth mindset](https://en.wikipedia.org/wiki/Mindset#Theories)
- Avoid being perfect, focus on incorporating evidence-based changes
- Have contributed to Human-GEM with at least **2** successfully merged Pull Requests
